src/Main.java

```diff
@@ -1,5 +1,127 @@
+        int weightIceCreamForRecipe = briquetteIceCream * gramInOneBriquetteIceCream;
+        byte egg = 4;
+        byte gramInOneEgg = 70;
+        int weightEggForRecipe = egg * gramInOneEgg;
+        int gramInOneKilogramm = 1000;
+        float weightRecipeInGram = weightBananaForRecipe + weightMilkForRecipe + weightIceCreamForRecipe + weightEggForRecipe;
+        float weightRecipeInKilogram = weightRecipeInGram / gramInOneKilogramm;
+        System.out.println("\nЗадача 6");
+        System.out.println("Вес спортзавтрака составляет " + weightRecipeInKilogram + " кг.");
+
+        byte weightLossInKilogram = 7; //начало задачи 7
+        short weightLossGramInDayMin = 250;
+        short weightLossGramInDayMax = 500;
+        int weightLossInGramm = weightLossInKilogram * gramInOneKilogramm;
+        int daysForWeightLossMax = weightLossInGramm / weightLossGramInDayMin;
+        int daysForWeightLossMin = weightLossInGramm / weightLossGramInDayMax;
+        int daysForWeightLossMean = (daysForWeightLossMin + daysForWeightLossMax) / 2;
+        System.out.println("\nЗадача 7");
+        System.out.println("На похудение уйдёт " + daysForWeightLossMax + " дней, если спортсмен будет терять каждый день по " + weightLossGramInDayMin + " грамм");
+        System.out.println("На похудение уйдёт " + daysForWeightLossMin + " дней, если спортсмен будет терять каждый день по " + weightLossGramInDayMax + " грамм");
+        System.out.println("В среднем на похудение уйдёт " + daysForWeightLossMean + " дней");
+
+        int salaryWorker1InMonth = 67760; //начало задачи 8
+        int salaryWorker2InMonth = 83690;
+        int salaryWorker3InMonth = 76230;
+        short monthInYear = 12;
+        byte indexingProperty = 10;
+        byte totalPercent = 100;
+
+        int salaryWorker1InYear = salaryWorker1InMonth * monthInYear;
+        int salaryWorker2InYear = salaryWorker2InMonth * monthInYear;
+        int salaryWorker3InYear = salaryWorker3InMonth * monthInYear;
+
+        int salaryWorker1InMonthAfterIndexing = salaryWorker1InMonth + (salaryWorker1InMonth*indexingProperty/totalPercent);
+        int salaryWorker2InMonthAfterIndexing = salaryWorker2InMonth + (salaryWorker2InMonth*indexingProperty/totalPercent);
+        int salaryWorker3InMonthAfterIndexing = salaryWorker3InMonth + (salaryWorker3InMonth*indexingProperty/totalPercent);
+
+        int salaryWorker1InYearAfterIndexing = salaryWorker1InMonthAfterIndexing * monthInYear;
+        int salaryWorker2InYearAfterIndexing = salaryWorker2InMonthAfterIndexing * monthInYear;
+        int salaryWorker3InYearAfterIndexing = salaryWorker3InMonthAfterIndexing * monthInYear;
+
+        int differenceSalaryWorker1InYear = salaryWorker1InYearAfterIndexing % salaryWorker1InYear;
+        int differenceSalaryWorker2InYear = salaryWorker2InYearAfterIndexing % salaryWorker2InYear;
+        int differenceSalaryWorker3InYear = salaryWorker3InYearAfterIndexing % salaryWorker3InYear;
```

Comment:
```suggestion
        int differenceSalaryWorker1InYear = salaryWorker1InYearAfterIndexing - salaryWorker1InYear;
        int differenceSalaryWorker2InYear = salaryWorker2InYearAfterIndexing - salaryWorker2InYear;
        int differenceSalaryWorker3InYear = salaryWorker3InYearAfterIndexing - salaryWorker3InYear;
```
остаток от деления не всегда будет правильно показывать разницу, поэтому лучше его не использовать. Например, 7 % 5 = 2 = 7 - 5, но 14 % 5 = 4 != 14 - 5 = 9

Reply:
согласен, тоже об этом думал, но в конкретной задаче рассматривалась зарплата и индексация, чтобы результат получился неверным, надо чтобы индексация была на 100% и более )))

Reply:
Ну да)